### PR TITLE
Remove cursor

### LINKED ISSUE #49: Remove cursor when leaving a document
Right now if the client is editing a document, then goes back to the documents page and then select the same document, the old cursor remains. We should add a `beforeDestroy` hook and remove the old Cursor.

### PR DESCRIPTION
**Pull request by viktor992**
Added a _beforeDestroy_ hook in editor.vue for removing the current user cursor. Also, a _remove_ method was added to _lib/documents/cursor.js_ and _server/documents/cursor.js_

Fixes #49

### CODE REVIEW

**Comment by viktor992** [comment]
Ready for review

File: server/documents/cursor.js [hunk 1]
```diff
@@ -7,6 +7,18 @@ import randomColor from 'randomcolor';
...
 Meteor.methods({
+
+  'Cursor.remove'(args) {
+    check(args, {
+      contentKey: Match.DocumentId,
+      clientId: Match.DocumentId,
+    });
+    Cursor.documents.remove({
+      contentKey: args.contentKey,
+      clientId: args.clientId,
```
Comment (mitar) [inline server/documents/cursor.js]: I would add also `connectionId: this.connection.id` as a condition.

Comment (viktor992) [reply server/documents/cursor.js]: Condition added.

**Comment by mitar** [comment]
Thanks.

**Pull request merged**